NSelene/SeleneDriver.cs

```diff
@@ -192,6 +192,11 @@ public SeleneDriver Should(Condition<IWebDriver> condition)
             Selene.WaitFor(this.Value, condition);
             return this;
         }
+        public SeleneDriver ShouldNot(Condition<IWebDriver> condition)
+        {
+            Selene.WaitForNot(this.Value, condition);
+            return this;
+        }
```

Comment:
We can keep it for now, but then, for simplicity, it would be better to have only one assert method - Should
and implement the "negation" – on the condition side...
we can have separate Not method to negate conditions, and add corresponding aliases to Be.Not.* (like Be.Not.Visible) and Have.No.* (like Have.No.CssClass - this one we already have)

Reply:
Nice idea, makes it intuitive

Reply:
I like fluent assertions a lot, maybe have a look at those too: https://fluentassertions.com/introduction

Reply:
I will have a look, yes:)

Reply:
https://github.com/yashaka/NSelene/issues/52